en/docs/integration-guides/usecases/http/build-an-http-rest-api-for-hotel-reservation.md

```
@@ -0,0 +1,345 @@
+<a href="{{base_path}}/assets/usecases/http-reservation/img/edit_cancel_resources.gif"><img src="{{base_path}}/assets/usecases/http-reservation/img/edit_cancel_resources.gif" alt="Edit and Cancel Resources" width="70%"></a>
+
+---
+
+## Step 7: Run and test the API
+
+1. Click the **Run** button in the BI extension.
+2. Wait for the service to start on port 8290.
+
+### Test the complete workflow
+
+**1. Start at the entry point**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)